### PR TITLE
Change from react-testing-library to testing-library/react

### PR DESCRIPTION
**Pull request by LeviButcher**
As requested in #59 

Let me know of anything else needed for this merge.

### CODE REVIEW

**Comment by LeviButcher** [comment]
@itaditya I've updated my branch to resolve conflicts with master. I forced push to prevent from creating many commits.

I realized the format command isn't running on the tests directory. Would it be okay to change it to
"prettier --write src/* src/**/*"?

**Comment by itaditya** [comment]
That's a good catch @LeviButcher. Ideally a PR should focus only on one thing. Since prettier formatting is different from your work.

**But for now, you can keep the prettier change**.

File: package.json [hunk 1]
```diff
@@ -10,18 +10,18 @@
...
-    "react-testing-library": "5.2.1"
+    "react-scripts": "1.1.4"
   },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject",
-    "format": "prettier --write src/*"
+    "format": "prettier --write src/* src/**/*"
```
Comment (itaditya) [inline package.json]:
```suggestion
    "format": "prettier --write src/**/*"
```

just doing this must be enough

Comment (LeviButcher) [reply package.json]: I'm on Ubuntu and having just src/**/* only found the tests directory. Wrapping it in single quotes seems to do the trick for getting everything.

**Comment by LeviButcher** [comment]
@itaditya I've updated my branch with latest changes.
The format command was breaking trying to format the mp3 file so I changed it to match only .js.
But that's not a very good solution since we probably want css files formatted as well.

I can add in a css file match but want to make sure your fine with that and if there are any other patterns we should match on.

**Comment by itaditya** [comment]
@LeviButcher prettier has the option to ignore files as well. See this https://prettier.io/docs/en/ignore.html#ignoring-files

So we can say ignore mp3 file via .prettierignore file

**Comment by LeviButcher** [comment]
@itaditya  Good to go. :+1:

File: .prettierignore [hunk 2]
```diff
@@ -0,0 +1 @@
+*.mp3
```
Comment (itaditya) [inline .prettierignore]: Add a newline at the end to make the red symbol go away.

Also configure your editor to add a newline to files automatically

**Comment by itaditya** [comment]
Can you run these commands-

```
git pull upstream master # get the latest code of my master in your fork's branch
npm install # install the dependencies so it auto-generates correct package-lock.json
```

This will automatically fix the merge conflicts.

**Comment by itaditya** [comment]
Congrats @LeviButcher

**Pull request merged**